app/lib/dartdoc/backend.dart

```diff
@@ -320,3 +320,58 @@ class DartdocBackend {
     await deletePool.close();
   }
 }
+
+class ArchiveTracker {
```

Comment:
There's no tests of this class, but perhaps this is temporary code? There's no documentation for this class. It's a bit unclear at first exactly what it is measuring.

Reply:
Yes, only temp code here. It just needs to give us a rough estimate on how much local disk space we'd need, and be able to estimate roughly how much network latency we would incur when downloading the package archives. Adding comment.